ketohub/spiders/raw_content_spider.py

```diff
@@ -1,4 +1,6 @@
+import json
 import os
+import re
 import urllib
 
 from datetime import datetime
```

Comment:
`datetime.datetime` is actually a class so this violates the style guide's rule about not importing classes

https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports